perceval/runtime/_token_management.py

```diff
@@ -101,6 +101,7 @@ def force_token(token: str):
         TokenProvider._CACHED_TOKEN = token
 
 
+@deprecated(version="0.13.0", reason="Use instead RemoteConfig class method `save`")
 def save_token(token: str):
     """Save provided token into persistent data, replace any token previously saved
```

Comment:
I think this should replace the token in the `RemoteConfig` too, since it's the one that will be used by `RemoteProcessor`